payments/credit-card/index.md

```diff
@@ -0,0 +1,694 @@
+---
+title: Swedbank Pay Payments Credit Card
+sidebar:
+  navigation:
+  - title: Payments
+    items:
+    - url: /payments/
+      title: Introduction
+    - url: /payments/credit-account
+      title: Credit Account Payments
+    - url: /payments/credit-card
+      title: Credit Card Payments
+    - url: /payments/credit-card/after-payment
+      title: Credit Card After Payments
+    - url: /payments/invoice
+      title: Invoice Payments
+    - url: /payments/direct-debit
+      title: Direct Debit Payments
+    - url: /payments/mobile-pay
+      title: Mobile Pay Payments
+    - url: /payments/swish
+      title: Swish Payments
+    - url: /payments/vipps
+      title: Vipps Payments
+---
+
+{% include alert.html type="warning"
+                      icon="warning"
+                      header="Site under development"
+                      body="The Developer Portal is under construction and should not be used to integrate against Swedbank Pay's APIs yet." %}
+
+
+>The basic redirect purchase scenario is the most common way to implement card payments.
+
+## Introduction
+
+* When properly set up in your merchant/webshop site and the payer starts the purchase process, you need to make a POST request towards PayEx with your Purchase information. This will generate a payment object with a unique paymentID. You either receive a Redirect URL to a hosted page or a JavaScript source in response.
+* You need to [redirect] the payer's browser to that specified URL, or embed the script source on your site to create a Hosted View in an iFrame; so that she can enter the credit card details in a secure PayEx hosted environment.
+* PayEx will handle 3D-secure authentication when this is required.
+* PayEx will redirect the payer's browser to - or display directly in the iFrame - one of two specified URLs, depending on whether the payment session is followed through completely or cancelled beforehand. Please note that both a successful and rejected payment reach completion, in contrast to a cancelled payment.
+* When you detect that the payer reach your completeUrl , you need to do a `GET` request to receive the state of the transaction, containing the paymentID generated in the first step, to receive the state of the transaction.
+
+## Screenshots
+
+You will redirect the payer to PayEx hosted pages to collect the credit card information.
+
+![Screnshot-1]
```

Comment:
Is this valid Markdown syntax? Shouldn't it be `![Alternative Text][footnote-url-reference-to-image]`?

Reply:
It's valid. 
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links Example 6

Reply:
But this isn't a link, it's an image. I can't find that short-hand syntax [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images).

Reply:
"It works on my machine" as one says. I can show it if you want, but it can also be changed.

Reply:
Ok, I believe you. 😄 But let's change it to the variant with alternative text, as we should provide that anyway.